src/billomat-resource-client.ts

```diff
@@ -53,8 +53,9 @@ export class BillomatResourceClient<T extends Billomat.Resource> {
                 reject('Unsupported resource (no singular defined)');
                 return;
             }
-            const payload = {};
-            Object.defineProperty(payload, singular, resource);
```

Comment:
I think I ran into some trouble here which is why I had to use `Object.defineProperty`, but I can't see why we'd still need it, so thanks for cleaning this up!